src/javascript/app_2/App/Components/Form/InputField/input-field.jsx

```diff
@@ -5,9 +5,9 @@ import {
 import PropTypes                 from 'prop-types';
 import React                     from 'react';
 import { isCryptocurrency }      from '_common/base/currency_base';
+import Tooltip                   from 'App/Components/Elements/tooltip.jsx';
```

Comment:
i think we can remove tooltip at all right?

Reply:
no, the way the app loads causes a bug at first load where the tooltip doesn't appear.